src/includes.hpp

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022, KNS Group LLC (YADRO)
+
+#pragma once
+
+#include "server.hpp"
```

Comment:
Likely this should be also wrapped by angle brackets.
Or all local headers should be in quotes.
The mix looks weird.

Reply:
Done